pkg/bgpv1/test/neighbor_test.go

```diff
@@ -173,7 +174,17 @@ func Test_NeighborAddDel(t *testing.T) {
 						holdTimeSeconds: peer.AppliedHoldTimeSeconds,
 					})
 				}
-				return assert.ElementsMatch(t, step.expectedPeerStates, runningState, step.description)
+				sort.Slice(step.expectedPeerStates, func(i, j int) bool {
```

Comment:
Consider moving this check to separate method, so this check can be reused by other tests.

IsPeerStateEqual(expected, running) bool

Reply:
good idea, done